File: Hyperdrive.Portal/Hyperdrive.Tier.Web/ClientApp/src/viewmodels/views/viewapplicationuserarchive.ts
import { ViewBase } from './viewbase';
import { ViewKey } from './viewkey';
import { ViewArchive } from './viewarchive';
import { ViewApplicationUser } from './viewapplicationuser';

export interface ViewApplicationUserArchive extends ViewKey, ViewBase {
    Archive: ViewArchive;
    ApplicationUser: ViewApplicationUser;
}

File: Hyperdrive.Service/Hyperdrive.Tier.ViewModels/Classes/Views/ViewApplicationUserArchive.cs
﻿using System;
using System.Xml.Serialization;

using Hyperdrive.Tier.ViewModels.Interfaces.Views;

namespace Hyperdrive.Tier.ViewModels.Classes.Views
{
    [XmlRoot("application-user-archive")]
    public class ViewApplicationUserArchive : IViewBase, IViewKey
    {
        public ViewApplicationUserArchive()
        {
        }

        [XmlElement("id")]
        public int Id { get; set; }

        [XmlElement("last-modified")]
        public DateTime LastModified { get; set; }

        [XmlElement("archive")]
        public virtual ViewArchive Archive { get; set; }

        [XmlElement("application-user")]
        public virtual ViewApplicationUser ApplicationUser { get; set; }
    }
}
